### PR TITLE
test: change all fixtures and add 'bragi' tests on API endpoints

### PR DESCRIPTION
**Pull request by woshilapin**
This PR is a lot, but is hard to separate. Here is what it does:
- remove all existing fixtures and replace them with a coherent set of fixtures around Corse (POI, OSM, BANO, OpenAddress, NTFS, cosmogony).
- `cosmogony` fixtures is automatically created from `corse.osm.pbf` and not committed into the repository
- update all tests with the new fixtures (keep an eye out, no test should be removed, no assertion either)
- add 3 tests for `bragi` (on each important API endpoint: `/autocomplete`, `/features` and `/reverse`)

### CODE REVIEW

File: benches/indexing.rs [hunk 1]
```diff
@@ -27,11 +27,11 @@ fn bench(c: &mut Criterion) {
             .await
             .expect("could not establish connection with Elasticsearch");
 
-        download::osm("ile-de-france").await.unwrap();
+        download::osm("corse").await.unwrap();
         // false: don't force regenerate admins for 'ile-de-france'
         cosmogony::generate("ile-de-france", false).await.unwrap();
         // true: force reindex admins on bench dataset for 'ile-de-france'
```
Comment (pbench) [inline benches/indexing.rs:33]:
```suggestion
        // false: don't force regenerate admins for 'corse'
        cosmogony::generate("ile-de-france", false).await.unwrap();
        // true: force reindex admins on bench dataset for 'corse'
```

**Review by azime — APPROVED** [review]

File: benches/listing.rs [hunk 2]
```diff
@@ -29,11 +29,11 @@ fn bench(c: &mut Criterion) {
             .await
             .expect("could not establish connection with Elasticsearch");
 
-        download::osm("ile-de-france").await.unwrap();
+        download::osm("corse").await.unwrap();
         // false: don't force regenerate admins for 'ile-de-france'
         cosmogony::generate("ile-de-france", false).await.unwrap();
         // true: force reindex admins on bench dataset for 'ile-de-france'
-        cosmogony::index_admins(&client, "ile-de-france", "bench", true, true)
+        cosmogony::index_admins(&client, "corse", "bench", true, true)
```
Comment (pbench) [inline benches/listing.rs:36]: Same as above ? `ile-de-france` should become `corse` ?

File: src/bin/bano2mimir.rs [hunk 3]
```diff
@@ -144,24 +139,24 @@ mod tests {
...
+        assert_eq!(addresses.len(), 2);
 
         let addr1 = addresses
             .iter()
-            .find(|&addr| addr.name == "10 Place de la Mairie")
+            .find(|&addr| addr.name == "1 Cours Napoléon")
             .unwrap();
 
-        assert_eq!(addr1.id, "addr:1.378886;43.668175:10");
+        assert_eq!(addr1.id, "addr:8.736635;41.920063:1");
```
Comment (azime) [inline src/bin/bano2mimir.rs:149]: Can you add test:
`assert_eq!(addr1.label, "1 Cours Napoléon");`

File: benches/indexing.rs (hunk 1 above)
Comment (woshilapin) [reply benches/indexing.rs:33]: Yes, I'm actually wondering if we keep the `bench`. I don't even know if they're useful, working, or even compiling? I'm up for another opinion on that.

Comment (pbench) [reply benches/indexing.rs:33]: well if they are not used/compiled anywhere, and you don't see how useful they are, I'd say you can remove them

File: src/bragi/tests.rs [hunk 4]
```diff
@@ -108,3 +108,153 @@ async fn status() {
...
+    assert_eq!(geocoding.get("zone_type").unwrap(), &json!("city"));
+}
+
+#[serial]
+#[test(tokio::test)]
+async fn reverse() {
+    start_bragi().await;
+    cosmogony2mimir().await;
+    osm2mimir().await;
+    bano2mimir().await;
```
[the remark this answers is not in the document]
Comment (woshilapin) [reply src/bragi/tests.rs:212]: Not exactly, I'm redoing the indexation, but the ElasticSeach is launched only once for all the tests.

**Review by pbench — APPROVED** [review]

**Pull request merged**